src/status_im/ui/screens/accounts/models.cljs

```diff
@@ -44,29 +45,33 @@
 
 (defn- add-account
   "Takes db and new account, creates map of effects describing adding account to database and realm"
-  [{:networks/keys [networks] :as db} {:keys [address] :as account}]
-  (let [enriched-account (assoc account
+  [{:keys [address] :as account} cofx]
+  (let [db (:db cofx)
+        {:networks/keys [networks]} db
+        enriched-account (assoc account
                                 :network config/default-network
                                 :networks networks
                                 :address address)]
     {:db                 (assoc-in db [:accounts/accounts address] enriched-account)
      :data-store/base-tx [(accounts-store/save-account-tx enriched-account)]}))
 
-(defn on-account-created [{:keys [pubkey address mnemonic]} password {:keys [signing-phrase status db]}]
+(defn on-account-created [{:keys [pubkey address mnemonic]} password seed-backed-up {:keys [signing-phrase status db] :as cofx}]
```

Comment:
Why do we change signature in breaking way? As opposed to new fn or possibly adding it to map (open, doesn't change positional args)

Reply:
it's a (or rather should be) a private method, so why not? :)

Reply:
If it should be a private fn we should probably indicate this with `defn-`

Reply:
I agree. We will have a follow-up PR for this issue, @yenda will take over. This one fixes the issue and reduces code duplication, but for sure it could be improved further.

Reply:
It can't be a private `defn-` because it's used from the events namespace and should be used from test namespaces. Speaking of unit tests, imo that's really the only way to address things like breaking interface changes. 

Things like using maps solve some of the issues we with too many positional args, but at the expense of reduced visibility of those params, i.e. without a spec there is no named args declaration, with specs it's hidden away.

Reply:
Luckily it's just a regular pure fn, so it can be unit tested just fine.